app/src/main/res/layout/content_settings_new_other.xml

```diff
@@ -37,7 +37,7 @@
         android:id="@+id/shareFeedbackSetting"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        app:primaryText="@string/leaveFeedback"
+        app:primaryText="@string/sendFeedback"
```

Comment:
Since this is not part of DuckChat, we should leave this string out and only add it once translations are complete.